.github/workflows/rector.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  push:
```

Comment:
Should it be used for pull requests too, like other workflows https://github.com/yiisoft/package-template/blob/852e46eebbf89dd157404a5b27b1d070e79fb194/.github/workflows/build.yml#L2, or it's unnecessary in this case?

Reply:
The reason is not call rector twice to prevent duplicating changes

Reply:
OK, got it.